Sweetener.Logging/LogLevel.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace Sweetener.Logging
+{
+    /// <summary>
+    /// Specifies a priority level and semantics associated with a log entry.
+    /// </summary>
+    public enum LogLevel
+    {
+        /// <summary>
+        /// Specifies often verbose debugging information used to diagnose potential problems.
+        /// </summary>
+        Debug,
+
+        /// <summary>
+        /// Specifies standard operating information used to monitor the state of an application.
+        /// </summary>
+        Info,
+
+        /// <summary>
+        /// Specifies problematic, but not exceptional, information.
+        /// </summary>
+        Warn,
```

Comment:
I feel like warning would be better here

Reply:
I understand this sentiment. The levels should be seemingly be nouns as if to say, "This message is a warning while this other message is an error." That makes fatal a... fatality? lol

The jury seems out on this. [Log4J 2.x Level](https://logging.apache.org/log4j/2.0/log4j-api/apidocs/org/apache/logging/log4j/Level.html) uses `warn`, but ASP.NET Core's `Microsoft.Extensions.Logging` has a [LogLevel](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=aspnetcore-2.2) enum with `warning`.
-